Remotion/Core/Tools/Console/ConsoleApplication/ConsoleApplication.cs

```diff
@@ -140,7 +140,7 @@ public virtual int RunApplication (TApplicationSettings settings)
         var application = CreateApplication();
         application.Run (settings, _errorWriter, _logWriter);
       }
-      catch (Exception e)
+      catch (Exception? e)
```

Comment:
that's a new one here, the nullable exeption in the catch block.

Reply:
It looks weird but I annotated it because of the reassignment to `e` in this block, a separate variable might be better here but more of a change.

Reply:
ah, yes. indeed. todo-comment + ticket, please.